src/main/java/ch/njol/skript/effects/EffBlockUpdate.java

```diff
@@ -0,0 +1,89 @@
+package ch.njol.skript.effects;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Effect;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.util.Kleenean;
+import org.bukkit.block.Block;
+import org.bukkit.block.BlockState;
+import org.bukkit.block.data.BlockData;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+@Name("BlockState - Update")
+@Description({
+	"Updates the blockstate of a block to cached state or selected block",
+	"`force`: Will force the update of the block",
+	"`without physics`: Does not send updates to surrounding blocks"
+})
+@Examples({
+	"set {_state} to blockstate of event-block",
+	"set event-block to air",
+	"wait 1 minute",
+	"force update {_state} without physics updates",
+	"",
+	"force update event-block as stone without physics updates"
+})
+@Since("INSERT VERSION")
+// Ported over from SkBee made by ShaneBee (Credits go to him)
+public class EffBlockUpdate extends Effect {
+
+	static {
+		Skript.registerEffect(EffBlockUpdate.class,
+			"[:force] update %blockstates% [physics:without (neighbour|physics) updates]",
+			"[:force] update %blocks% as %blockdata% [physics:without (neighbour|physics) updates]");
```

Comment:
How about adding `adjacent` as pattern and/or making `physics` optionnal? I'm not English-native but I thought it would be intuitive.
```suggestion
			"[:force] update %blockstates% [physics:without [(neighbour|adjacent)] [physics] updates]",
			"[:force] update %blocks% as %blockdata% [physics:without [(neighbour|adjacent)] [physics] updates]");
```